dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/authentication/JakartaOidcAuthorizationRequest.java

```diff
@@ -38,6 +41,10 @@ public class JakartaOidcAuthorizationRequest extends AuthorizationRequest {
 
     public static final TraceComponent tc = Tr.register(JakartaOidcAuthorizationRequest.class);
 
+    public static final String STORED_REQUEST_METHOD = "STORED_REQUEST_METHOD";
```

Comment:
In a follow up PR, we should add a hash based on the "state" to avoid collisions.